app/serializers/api/v2/csv/heading_serializer.rb

```diff
@@ -0,0 +1,24 @@
+module Api
+  module V2
+    module Csv
+      class HeadingSerializer
+        include CsvSerializer
+
+        columns :goods_nomenclature_item_id,
+                :goods_nomenclature_sid,
+                :declarable,
+                :description,
+                :description_plain,
+                :formatted_description,
+                :producline_suffix
+
+        column :leaf do |heading|
```

Comment:
This should just be:

```ruby
heading.commodities.any?
```

Reply:
yeh, it works
dataset provides also the method `empty?`, which just looks nicer: `heading.commodities_dataset.empty?` 
https://www.rubydoc.info/github/jeremyevans/sequel/Sequel%2FDataset:empty%3F